packages/headless/src/app/logger.ts

```diff
@@ -11,7 +11,7 @@ export interface LoggerOptions {
    * Changes the shape of the log object. This function will be called every time one of the log methods (such as `.info`) is called.
    * All arguments passed to the log method, except the message, will be pass to this function. By default it does not change the shape of the log object.
    */
-  logFormatter?: (object: object) => object;
+  logFormatter?: (object: Record<string, unknown>) => Record<string, unknown>;
```

Comment:
🤔 https://www.typescriptlang.org/play?#code/C4TwDgpgBAglC8UBKEDGB7ATgEwDwGdhMBLAOwHMAaAV1IGtT0B3UgPgG4AobNAGwENM0DKUJQAZunQAudACMAVmmBQuk9FHz9gxfOOIR8sdkA
Technically breaking change, `{}` does not satisfies `Record<string, unknown>`.

Reply:
We do end up exporting the logger, and so you are right that this is a breaking change.

However in this case I'm "fairly certain" this would have no impact/no one relies on that formatter.

I'm curious why we need to change it though @y-lakhdar  ? 🤔

Reply:
In the latest version of pino, the log property accepts a Record<string, unknown> only  and...
```
Type 'object' is not assignable to type 'Record<string, unknown>'.
```

I'll check to see if we can avoid doing a breaking change by using `{}` instead since it is accepted by pino

Reply:
Chances of this actually causing issues seems pretty low, and fix looks like it'll be intuitive in most cases, so YOLO I think

Reply:
(to be clear, it won't keep me up at night, I just flagged the thing just in case)